src/polyglot-notebooks-vscode/do-version-upgrade.ps1

```diff
@@ -36,6 +36,12 @@ try {
     Remove-Item -Path "$stableDirectory\src\*" -Filter "*.ts"
     Copy-Item -Path "$insidersDirectory\src\*" -Destination "$stableDirectory\src\" -Filter "*.ts"
 
+    # copy localization files
```

Comment:
Quick question: So if we have this in the script, do we still need the package.nls.*.json files below? Looks like they exist here: https://github.com/dotnet/interactive/tree/main/src/polyglot-notebooks-vscode-insiders

Reply:
package.nls.*.json should override the existing ones to keep in sync with the ones in insiders